### PR TITLE
landing page: redesign contributors section

### LINKED ISSUE #1037: landing page: update design for contributors section
Update according to mockups (ref #988)

- Group all the contributor-types, and display the groups horizontally instead of vertically to avoid too much whitespace, and to make it easier to go through the related content without scrolling and losing context.
- Have only one "show affiliations"-button if they're all showing the same content (it's not necessary to have more than one button for the same thing).
- Move "show-affiliations"-button to the very right, bottom of the section to indicate that the extra content is relevant for all the groups, and that it will show up below.

### PR DESCRIPTION
**Pull request by jennur**
closes #1037

Updated design according to mockups:

Screenshot 2021-08-03 at 18 52 47

Desktop:
Screenshot 2021-08-03 at 18 25 35

Tablet:
Screenshot 2021-08-03 at 18 25 15

Mobile:
Screenshot 2021-08-03 at 18 25 26

### CODE REVIEW

**Review by ppanero — APPROVED** [review]
LGTM, Very nice work! just waiting for LTS release before merging.

**Comment by ppanero** [comment]
Picky comment: Forgot to mention, in the desktop version... when the list of names is not too large, could the button be shown in the same line? it looks a bit weird to have it in the next line but near (it feels like something is wrongly displayed). If we want to put it in the corner maybe we can find a way of showing that it is actually in the corner not an overflow line?

Screenshot 2021-08-04 at 10 32 18

File: invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/creatibutors.html
Comment (ppanero) [reply invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/creatibutors.html:17]: Discussed IRL: My mistake the `i` tag is for groups (organization) not for the `ror` one. That is below and it actually uses the same css clases.

**Comment by jennur** [comment]
Testing with more space between content and button. Let me know what you think? I think the button disappears a little if it's right next to the content?

Screenshot 2021-08-05 at 10 33 22

**Comment by jennur** [comment]
### Suggestions
Here are some new design suggestions after review & discussions. Some of the discussed topics:

- The previous suggestion emphasises the contributors a lot (too much), and makes the creators look less important. 
- Having only one show affiliations-button is cleaner (and it's not necessary to have more than one). 

**Note:** I did not update the PR with these changes yet as they require the functionality proposed in [#802](https://github.com/inveniosoftware/invenio-rdm-records/issues/802). 

contributors

**Affiliations**

affiliations

**In context**

Screenshot 2021-08-20 at 19 25 56
Screenshot 2021-08-20 at 19 26 05

**Comment by jennur** [comment]
This PR/branch is included in #1099

**Comment by lnielsen** [comment]
Integrated in #1099

**Pull request closed without merging**
